README.md

```diff
@@ -1,2 +1,3 @@
 # weraWork
 GitHub con la Güera
+Esto no me gusta nada!  Vamos a cambiar este README.
```

Comment:
_ValentinaPV:_ Do you think the exclamation point in your change is absolutely necessary?